components/Business.vue

```diff
@@ -57,7 +57,7 @@
               <client-only>
                 <v-icon name="map-marker" class="icon" />
               </client-only>
-              <span>{{ business.address }}</span>
+              <span>{{ business.address }}, {{ business.city }}</span>
```

Comment:
I wonder if this should be conditional based on which region is embedding the directory, as presumably in the London embed it is always going to be 'London' and redundant.